taxi/models.py

```diff
@@ -0,0 +1,20 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):

```

Comment:
The `Meta` class is missing in the `Manufacturer` model. According to the checklist, you should always implement `Meta` class after you defined all attributes. The `Meta` class should contain `verbose_name` and `verbose_name_plural` in lowercase.